plugins/modules/net_tools/nmcli.py

```diff
@@ -67,6 +67,12 @@
         type: str
         choices: [ 802.3ad, active-backup, balance-alb, balance-rr, balance-tlb, balance-xor, broadcast ]
         default: balance-rr
+    transport_mode:
+        description:
+            - This option sets the connection type of Infiniband IPoIB devices.
+        type: str
+        choices: [ datagram, connected ]
+        default: datagram
```

Comment:
It's usually best *not* to set a default, since having this default is potentially a breaking change for folks who already use this connection type with `transport_mode=connected` (when they have set that value not with this module).